ios/Podfile.lock

```diff
@@ -0,0 +1,137 @@
+PODS:
```

Comment:
Also please explain what this file does too

Reply:
Lock files are generally generated files that list dependency versions (all our libraries listed with their versions)